README.md

```diff
@@ -178,6 +179,14 @@ Nesting is also supported:
 }
 ```
 
+A variable may also be retrieved for an explicit variant using the `$force-variant` argument of `oBrandGet`. This is useful when retrieving a variant value within a function, or when a specific value is needed regardless of what variant is configured by `oBrandConfigureFor`.
+
+```scss
+	.o-example--inverse {
+		content: oBrandGet('o-example', 'component-content', $force-variant: 'inverse'); // "inverse" variant value
+	}
+```
+
 ### Output Styles Only If A Brand Supports A Variant
 
 Not all brands will share variants. Define support in the `settings` map as demonstrated above. To output styles only if the current brand supports the variant use `oBrandConfigureFor`.
```

Comment:
You said you'd update the README after this, but just noting that 'above' isn't applicable anymore